src/content/changes/highlights/2023-04-13.md

```diff
@@ -0,0 +1,14 @@
+---
+date: 2023-04-13T11:00:00
+title: Highlights for the week ending April 13, 2023
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+## General
+
+## Apps
+
+## Documentation
+
```

Comment:
Adding them here so I don't forget. They need to go into the Apps section (once Zach comment is added)
```suggestion
- [linkerd-bundle](https://github.com/giantswarm/linkerd-bundle) is now generally available from the `giantswarm` catalog
- [aws-load-balancer-controller](https://github.com/giantswarm/aws-load-balancer-controller-app) version [1.3.0](https://github.com/giantswarm/aws-load-balancer-controller-app/blob/main/CHANGELOG.md#130---2023-04-12) automatically sets the annotations needed for AWS authentication with IRSA.
```